WordPress-Theme/ruleset.xml

```diff
@@ -3,7 +3,7 @@
 	<!-- For more information: https://make.wordpress.org/themes/handbook/review/ -->
 	<description>Standards any Theme to be published on wordpress.org should comply with.</description>
 
-
+	<rule ref="WordPress.Functions.FunctionRestrictions"/>
```

Comment:
I'd rather you don't.
It'll be so much easier to manage the project if every rule has their own sniff. If a rule changes, we change the sniff without potentially impacting other sniffs.

Also, having each rule in a separate sniff will also allow for more modular disabling of sniffs.
While this won't be allowed for wp.org (of course!), I can imagine, commercial theme hosting parties which also have some form of theme review in place creating their own rule set based on the wp.org Theme ruleset and just selectively excluding some rules.